eng/helix.proj

```diff
@@ -0,0 +1,152 @@
+      <HelixPreCommands Condition="$(HelixTargetQueues.ToLowerInvariant().Contains('osx'))">$(HelixPreCommands);install_name_tool -change "/usr/local/opt/libomp/lib/libomp.dylib" "$HELIX_WORKITEM_ROOT/libomp.dylib" runtimes/osx-x64/native/libonnxruntime.dylib</HelixPreCommands>
+      <PublishFolder></PublishFolder>
+      <PublishFolder Condition="$(BuildConfig.EndsWith('-netfx'))">\win-$(BuildArchitecture)</PublishFolder>
+    </PropertyGroup>
+
+    <!-- Copy libiomp5.dylib and libomp.dylib to the publish folders for OSX-->
+    <Copy Condition="$(HelixTargetQueues.ToLowerInvariant().Contains('osx'))"
+      SourceFiles = "/usr/local/opt/libomp/lib/libiomp5.dylib;/usr/local/opt/libomp/lib/libomp.dylib;"
+      Retries="10"
+      OverwriteReadOnlyFiles="true"
+      RetryDelayMilliseconds="10"
+      DestinationFolder="$(BUILD_SOURCESDIRECTORY)\artifacts\bin\%(ProjectsWithTargetFramework.Filename)\$(BuildConfig)\%(ProjectsWithTargetFramework.TargetFrameworks)$(PublishFolder)">
+    </Copy>
+
+    <!-- Remove the native libraries for other OS to save on payload size -->
+    <ItemGroup>
+      <WindowsFiles Include="$(BUILD_SOURCESDIRECTORY)\artifacts\bin\%(ProjectsWithTargetFramework.Filename)\$(BuildConfig)\%(ProjectsWithTargetFramework.TargetFrameworks)$(PublishFolder)\runtimes\win*\**\*.*"/>
+      <LinuxFiles Include="$(BUILD_SOURCESDIRECTORY)\artifacts\bin\%(ProjectsWithTargetFramework.Filename)\$(BuildConfig)\%(ProjectsWithTargetFramework.TargetFrameworks)$(PublishFolder)\runtimes\linux*\**\*.*"/>
+      <OsxFiles Include="$(BUILD_SOURCESDIRECTORY)\artifacts\bin\%(ProjectsWithTargetFramework.Filename)\$(BuildConfig)\%(ProjectsWithTargetFramework.TargetFrameworks)$(PublishFolder)\runtimes\osx*\**\*.*"/>
+    </ItemGroup>
+
+    <Delete Condition="$(HelixTargetQueues.ToLowerInvariant().Contains('windows'))"
+      Files="@(LinuxFiles);@(OsxFiles)" />
+
+    <Delete Condition="$(HelixTargetQueues.ToLowerInvariant().Contains('osx'))"
+      Files="@(LinuxFiles);@(WindowsFiles)" />
+
+    <Delete Condition="$(HelixTargetQueues.ToLowerInvariant().Contains('ubuntu'))"
+      Files="@(WindowsFiles);@(OsxFiles)" />
+
+    <ItemGroup>
+      <HelixWorkItem Include="%(ProjectsWithTargetFramework.Filename)">
+        <PayloadDirectory>$(BUILD_SOURCESDIRECTORY)\artifacts\bin\%(ProjectsWithTargetFramework.Filename)\$(BuildConfig)\%(ProjectsWithTargetFramework.TargetFrameworks)\$(PublishFolder)</PayloadDirectory>
+        <Command Condition="$(IsPosixShell)">dotnet exec --roll-forward Major --runtimeconfig %(ProjectsWithTargetFramework.Filename).runtimeconfig.json --depsfile %(ProjectsWithTargetFramework.Filename).deps.json $HELIX_CORRELATION_PAYLOAD/xunit-runner/tools/netcoreapp2.0/xunit.console.dll %(ProjectsWithTargetFramework.Filename).dll -notrait Category=SkipInCI -xml testResults.xml</Command>
```

Comment:
NIT: 
You could avoid duplicating the command and just changing the env var access syntax if you capture the syntax in a property. i.e:

`<HelixCorrelationPayload Condition=IsPosix>%HELIX%</HelixCorrelationPayload>`

And then you can use that property.

Reply:
So `%HELIX%` maps to the correct path then? Let me try that and see if it works. I'll fix the spacing too.

Reply:
No, %HELIX% doesn't map to the correct path, that was just an example of how you could leverage the decision of the syntax you want to use with a property.

Reply:
Yeah, I figured that much when I took a second look. lol. But I got it correct now and only have the one command now.

I'm gonna go ahead and merge this in.